files/default/sql/ddl/updates/2.1.0.sql

```diff
@@ -54,13 +67,15 @@ CREATE TABLE IF NOT EXISTS `feature_store_connector` (
   `s3_id`                   INT(11),
   `hopsfs_id`               INT(11),
   `redshift_id`             INT(11),
+  `adls_id`                 INT(11),
```

Comment:
I think you forgot the undo for this column.

Reply:
Actually we don't need the undo for that one, as the `feature_store_connector` table was introduced in 2.1.0, so in the UNDO we already have:
```
DROP TABLE `feature_store_connector`;
```

Reply:
aaah, sorry I missed that.